style.css

```diff
@@ -0,0 +1,147 @@
+:root {
+  --color-main_theme: #36b37e;
+  --color-second_green: #c5e9da;
+  --color-light-green: #ebf0ee;
+  --color-dark-blue: #172b4d;
+  --color-white-bg: #e5e5e5;
+  --color-white: #fff;
+  --color-button-enabled: #36b37f;
+  --color-button-pressed: #008552;
+  --color-button-hover: rgba(54, 179, 127, 0.24);
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  font-size: 1.6rem;
+}
+
+.header {
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+header {
+  width: 100vw;
+  max-width: 800px;
+  height: 5vh;
+  min-height: 65px;
+  background-color: var(--color-white-bg);
+}
+
+nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+nav ul {
+  list-style: none;
+  display: none;
+}
+
+.burger {
+  display: flex;
+  width: 18px;
+  height: 12px;
+  margin-top: 25px;
+  margin-right: 14px;
+  flex-direction: column;
+  align-self: flex-start;
+  justify-content: space-around;
+  cursor: pointer;
+}
+
+.burger-line {
+  border-radius: 20px;
+  height: 2px;
+  background-color: var(--color-dark-blue);
+}
+
+/* ----- MAIN SECTION ----- */
+
+#main {
+  background-color: var(--color-light-green);
+  display: flex;
+  justify-content: center;
+}
+
+.main {
+  max-width: 800px;
+  max-height: 800px;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: space-around;
+  align-items: center;
+  background-image: url("./assets/img/header-bg-mobile.png");
```

Comment:
- [OPTIONAL] I think it would be much better to use `.svg` instead of `.png` 💪 While `pngs` are capable of handling very high resolutions, they're not infinitely expandable. On the other hand, `svg` files can expand to any size without losing their resolution ℹ